README.md

```diff
@@ -49,6 +49,7 @@ Base repository: [golang](https://hub.docker.com/_/golang)
 
 ### Tags
 
+- [1.23-alpine](https://github.com/loft-sh/devspace-containers/pkgs/container/devspace-containers%2Fgo/206002054?tag=1.23-alpine) (Base image: [golang:1.23-alpine]((https://hub.docker.com/_/golang?tab=tags&name=1.23-alpine)))
```

Comment:
That won't work, as GHCR only generates the ID for the URL once the image exists.
So, the tag will have to be updated in a follow-up PR.

```suggestion
```